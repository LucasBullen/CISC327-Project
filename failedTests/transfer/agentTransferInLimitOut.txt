1,14c1,14
< Select mode: atm or agent.
< Logged into agent mode.
< Enter account number to deposit to.
< Enter amount to deposit.
< Deposit successful.
< Enter first account number.
< Enter second account number.
< Enter amount to transfer.
< Transfer successful.
< Enter first account number.
< Enter second account number.
< Enter amount to transfer.
< Transfer successful.
< Logged out.
---
> Select mode: atm or agent.
> Logged into agent mode.
> Enter number of account to deposit into.
> Enter amount (in cents) to deposit.
> Deposit successful.
> Enter number of account to transfer from.
> Enter number of account to transfer to.
> Enter amount (in cents) to transfer.
> Trasfer successful.
> Enter number of account to transfer from.
> Enter number of account to transfer to.
> Enter amount (in cents) to transfer.
> Trasfer successful.
> Logged out.
