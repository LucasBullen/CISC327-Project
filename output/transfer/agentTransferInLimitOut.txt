Select mode: atm or agent.
Logged into agent mode.
Enter account number to deposit to.
Enter amount to deposit.
Deposit successful.
Enter first account number.
Enter second account number.
Enter amount to transfer.
Transfer successful.
Enter first account number.
Enter second account number.
Enter amount to transfer.
Transfer successful.
Logged out.
