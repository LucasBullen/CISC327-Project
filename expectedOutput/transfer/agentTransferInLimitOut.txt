Select mode: atm or agent.
Logged into agent mode.
Enter number of account to deposit into.
Enter amount (in cents) to deposit.
Deposit successful.
Enter number of account to deposit into.
Enter number of account to transfer to.
Enter amount (in cents) to transfer.
Trasfer successful.
Enter number of account to deposit into.
Enter number of account to transfer to.
Enter amount (in cents) to transfer.
Trasfer successful.
Logged out.
